test/StandardAssetRegistry.js

```diff
@@ -242,6 +242,56 @@ contract('StandardAssetRegistry', accounts => {
     })
   })
 
+  describe('Transacions Related', () => {
+    it('ensures the assets is pass to another owner after transfer', async () => {
+      const CONTENT_DATA = 'ramdom data'
+      const firstOwner = accounts[accounts.length - 2]
```

Comment:
please use the same name as defined on top https://github.com/decentraland/erc821/blob/c5290cb21dd9202ad5d60678debdd347bd47186f/test/StandardAssetRegistry.js#L61